ktor-client/ktor-client-core/js/src/io/ktor/client/engine/js/JsClientEngine.kt

```diff
@@ -129,7 +134,7 @@ private fun Event.asString(): String = buildString {
     append(JSON.stringify(this@asString, arrayOf("message", "target", "type", "isTrusted")))
 }
 
-private fun io.ktor.client.fetch.Headers.mapToKtor(): Headers = buildHeaders {
+private fun org.w3c.fetch.Headers.mapToKtor(): Headers = buildHeaders {
```

Comment:
Are you sure this works on nodeJS? See also https://github.com/ktorio/ktor/pull/2592

Reply:
In this case, we replace some types, and the actual implementation stays the same.

Reply:
The tests are also fine

Reply:
Unit tests? AFAIK they does not work with nodeJS production, at least 1 year ago, which results into the same problem at runtime: https://github.com/ktorio/ktor/pull/2385 so you needed to create a test project and use the artifact.

Reply:
I will try to set up a sample